src/base64.cr

```diff
@@ -153,7 +153,7 @@ module Base64
     strict_encode_to_io_internal(data, io, CHARS_SAFE, pad: true)
   end
 
-  # Returns the Base64-decoded version of `data` as a *Bytes*.
+  # Returns the Base64-decoded version of `data` as a `Bytes`.
```

Comment:
https://crystal-lang.org/docs/conventions/documenting_code.html says to use `*` for param names and `\`` for types, but if it's wrong throughout the file that should probably be a separate PR.